lib/ReactViews/ActionBar/ActionBarPortal.tsx

```diff
@@ -23,7 +23,7 @@ const ActionBarPortal: React.FC<PropsType> = ({ show }) => {
 const StyledPortal = styled(Portal)<{ show: boolean }>`
   display: flex;
   position: absolute;
-  height: 56px;
+  height: ${(p) => (p.show ? "56px" : "0")};
```

Comment:
this was necessary as element of screen was causing the whole page to move as scrollable on scrollIntoView

Reply:
FYI @na9da